packages/bitcore-node/src/modules/ethereum/api/csp.ts

```diff
@@ -13,15 +13,24 @@ import { SpentHeightIndicators } from '../../../types/Coin';
 import { EthListTransactionsStream } from './transform';
 import { ERC20Abi } from '../abi/erc20';
 import { Transaction } from 'web3/eth/types';
-import { EventLog } from 'web3/types';
```

Comment:
`EventLog` type was removed in `web3@1.2.4`

Reply:
https://github.com/ethereum/web3.js/blob/043cb81d40ede0f6ef1889222c5671f9678e409e/packages/web3/types/index.d.ts